docs/network/utxos.md

```diff
@@ -136,84 +142,9 @@ A user may also want to *split* UTXOs if one would like to exit an amount smalle
-
-<!--DOCUSAURUS_CODE_TABS-->
-<!-- JavaScript (ESNext) -->
-```js
-async function splitUtxo() {
-  // define payments objects that will represent new utxos
-  const payments = [
-    {
-      owner: "0x8CB0DE6206f459812525F2BA043b14155C2230C0",
-      currency: "0xd74ef52053204c9887df4a0e921b1ae024f6fe31",
-      amount: "120000000000000",
-    },
-    {
-      owner: "0x8CB0DE6206f459812525F2BA043b14155C2230C0",
-      currency: "0xd74ef52053204c9887df4a0e921b1ae024f6fe31",
-      amount: "250000000000000",
-    },
-    {
-      owner: "0x8CB0DE6206f459812525F2BA043b14155C2230C0",
-      currency: "0xd74ef52053204c9887df4a0e921b1ae024f6fe31",
-      amount: "250000000000000",
-    },
-  ];
-
-  // create a transaction body
-  const transactionBody = await childChain.createTransaction({
-    owner: "0x8CB0DE6206f459812525F2BA043b14155C2230C0",
-    payments,
-    fee: {
-      currency: "0xd74ef52053204c9887df4a0e921b1ae024f6fe31",
-    },
-  });
-
-  // sanitize transaction into the correct typedData format
-  // the second parameter is the address of the RootChain contract
-  const typedData = OmgUtil.transaction.getTypedData(
-    transactionBody.transactions[0],
-    "0x96d5d8bc539694e5fa1ec0dab0e6327ca9e680f9"
-  );
-
-  // define private keys to use for transaction signing
-  const privateKeys = new Array(
-    transactionBody.transactions[0].inputs.length
-  ).fill("0xCD55F2A7C476306B27315C7986BC50BD81DB4130D4B5CFD49E3EAF9ED1EDE4F7");
-
-  // locally sign typedData with passed private keys, useful for multiple different signatures
-  const signatures = childChain.signTransaction(typedData, privateKeys);
-
-  // return encoded and signed transaction ready to be submitted
-  const signedTxn = childChain.buildSignedTransaction(typedData, signatures);
-
-  // submit to the child chain
-  return childChain.submitTransaction(signedTxn);
-}
-
-```
-<!--END_DOCUSAURUS_CODE_TABS-->
-
-### Lifecycle
+In the above scenario, Alice can split her UTXO to withdraw 2 ETH.
 
-1. A user defines multiple `payments` objects that will generate new UTXOs. The sender is specified as the recipient.
-2. A user calls the `createTransaction` function to create a transaction that will split a single UTXO.
-3. A user signs, encodes, and submits the transaction's data to the child chain and the Watcher for validation.
-4. If the transaction is valid, the child chain server creates a transaction hash and adds the transaction to a pending block.
-5. The child chain bundles the transactions in the block into a Merkle tree and submits its root hash to the `Plasma Framework` contract.
-6. The Watcher receives a list of transactions from the child chain and recomputes the Merkle root to check for any inconsistency.
+Currently, UTXO splitting is not available via omg-js lib but you can use the OMG Network [Web Wallet](/environments#tools) to achieve the same result.
```

Comment:
this is available via omg-js/util/transaction:
https://github.com/omgnetwork/omg-js/blob/master/packages/omg-js-util/src/transaction.js#L226

but it does require for the user to create transaction manually without calling child chain /transaction.create